### PR TITLE
feat: Add EF

### PR DESCRIPTION
**Pull request by taustad**

### CODE REVIEW

File: datasheetapi/Models/DatabaseContext.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    }
+    public DbSet<Comment>? Comments { get; set; }
+    public DbSet<Contract>? Contracts { get; set; }
+    public DbSet<RevisionContainer>? RevisionContainers { get; set; }
+    public DbSet<RevisionContainerReview>? RevisionContainerReviews { get; set; }
+    public DbSet<TagDataReview>? TagDataReviews { get; set; }
+    public DbSet<Project>? Projects { get; set; }
+
+
+    // protected override void OnModelCreating(ModelBuilder modelBuilder)
```
Comment (DanielBohme) [inline datasheetapi/Models/DatabaseContext.cs:20]: WellProjectWell? Guess this is from ConceptApp? :D

File: datasheetapi/Services/TagDataReviewService.cs [hunk 2]
```diff
@@ -48,11 +48,10 @@ public async Task<List<TagDataReview>> GetTagDataReviewsForTags(List<Guid> tagId
 
     public async Task<TagDataReview> CreateTagDataReview(TagDataReview review, Guid azureUniqueId)
     {
-        var tagData = await _tagDataService.GetTagDataById(review.TagDataId) ?? throw new Exception($"Invalid tag data id: {review.TagDataId}");
+        var _ = await _tagDataService.GetTagDataById(review.TagDataId) ?? throw new Exception($"Invalid tag data id: {review.TagDataId}");
```
Comment (DanielBohme) [inline datasheetapi/Services/TagDataReviewService.cs:51]: weird variable name

File: tests/Services/RevisionContainerReviewServiceTests.cs [hunk 3]
```diff
@@ -146,67 +146,67 @@ public async Task GetRevisionContainerReviewDtosForTag_ReturnsDtos()
...
-        var tagData = new List<ITagData> { new TagData { RevisionContainer = new RevisionContainer { Id = Guid.NewGuid() } } };
-        var reviewModel = new RevisionContainerReview();
-        var savedReview = new RevisionContainerReview();
-        _tagDataServiceMock.Setup(x => x.GetAllTagData()).ReturnsAsync(tagData);
-        _reviewRepositoryMock.Setup(x => x.AddTagDataReview(reviewModel)).ReturnsAsync(savedReview);
-
-        // Act & Assert
-        await Assert.ThrowsAsync<Exception>(() => _reviewService.CreateRevisionContainerReview(review, azureUniqueId));
-    }
+    // [Fact]
```
Comment (DanielBohme) [inline tests/Services/RevisionContainerReviewServiceTests.cs:149]: Is this supposed to be commented out?

File: tests/Services/RevisionContainerReviewServiceTests.cs [hunk 4]
```diff
@@ -146,67 +146,67 @@ public async Task GetRevisionContainerReviewDtosForTag_ReturnsDtos()
...
-        var tagData = new List<ITagData> { new TagData { RevisionContainer = new RevisionContainer { Id = review.RevisionContainerId } } };
-        var reviewModel = new RevisionContainerReview();
-        var savedReview = new RevisionContainerReview();
-        _tagDataServiceMock.Setup(x => x.GetAllTagData()).ReturnsAsync(tagData);
-        _reviewRepositoryMock.Setup(x => x.AddTagDataReview(reviewModel)).ReturnsAsync(savedReview);
-
-        // Act & Assert
-        await Assert.ThrowsAsync<Exception>(() => _reviewService.CreateRevisionContainerReview(review, azureUniqueId));
-    }
+    // [Fact]
```
Comment (DanielBohme) [inline tests/Services/RevisionContainerReviewServiceTests.cs:177]: Is this supposed to be commented out?

File: datasheetapi/Services/TagDataReviewService.cs (hunk 2 above)
Comment (taustad) [reply datasheetapi/Services/TagDataReviewService.cs:51]: _ is a discard. I only use the method to check if the tag exists

**Review by DanielBohme — APPROVED** [review]
LGTM

**Pull request merged**